generator-prisma-client/deno-deploy/prisma.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'prisma/config'
+
```

Comment:
Should we add `import "dotenv/config";` here?

While trying to follow the instructions from README.md, I encountered this error:

<img width="679" height="314" alt="Screenshot 2025-08-07 at 5 02 52 PM" src="https://github.com/user-attachments/assets/989ff48f-58db-4d56-99b0-6bb25d170f17" />

I believe it is expected that env files are not loaded by default, but for this example we should explicitly load?